Firestore/Example/Tests/Local/FSTMutationQueueTests.mm

```diff
@@ -83,32 +83,30 @@ - (void)testCountBatches {
 - (void)testAcknowledgeBatchID {
   if ([self isTestBaseClass]) return;
 
-  // Initial state of an empty queue
-  XCTAssertEqual([self.mutationQueue highestAcknowledgedBatchID], kFSTBatchIDUnknown);
-
-  // Adding mutation batches should not change the highest acked batchID.
   self.persistence.run("testAcknowledgeBatchID", [&]() {
+    XCTAssertEqual([self batchCount], 0);
+
     FSTMutationBatch *batch1 = [self addMutationBatch];
     FSTMutationBatch *batch2 = [self addMutationBatch];
     FSTMutationBatch *batch3 = [self addMutationBatch];
     XCTAssertGreaterThan(batch1.batchID, kFSTBatchIDUnknown);
     XCTAssertGreaterThan(batch2.batchID, batch1.batchID);
     XCTAssertGreaterThan(batch3.batchID, batch2.batchID);
 
-    XCTAssertEqual([self.mutationQueue highestAcknowledgedBatchID], kFSTBatchIDUnknown);
+    XCTAssertEqual([self batchCount], 3);
```

Comment:
Checking `batchCount` is the only replacement for the previous check that I could think of.